### PR TITLE
[issue-251] - Missing Translations/Key - German/Japanese

### PR DESCRIPTION
**Pull request by DeeJayElly**
[issue-251] - Missing Translations/Key - German/Japanese

### CODE REVIEW

File: source/features/i18n/translations/404/ja.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-    "code": "!!!Error code:",
-    "description": "!!!The requested page cannot be found. It may have been removed or the link can be broken. If you entered a web address please check it was correct and try again.",
-    "explorerTitle": "!!!Blockchain Explorer",
-    "support": "!!!Contact Support",
-    "title": "!!!Page not found"
+    "code": "Error code:",
+    "description": "The requested page cannot be found. It may have been removed or the link can be broken. If you entered a web address please check it was correct and try again.",
+    "explorerTitle": "Blockchain Explorer",
+    "support": "Contact Support",
+    "title": "Page not found"
```
Comment (DominikGuzei) [inline source/features/i18n/translations/404/ja.json]: Should the error page not be Japenese?

Comment (DominikGuzei) [reply source/features/i18n/translations/404/ja.json]: Nevermind, i fixed it by merging-in the #248 issue into this one

[the remark this answers is not in the document]
Comment (DominikGuzei) [reply source/features/i18n/translations/404/ja.json]: No we need Japanese translations for the error page 👍 @DeeJayElly what's the status on that?

[the remark this answers is not in the document]
Comment (DeeJayElly) [reply source/features/i18n/translations/404/ja.json]: Someone reverted it. I fixed it now

**Comment by DeeJayElly** [comment]
> Translations for 404 page need to be provided for Japenese 🙏

Added

**Pull request merged**